modules/cache/mw.cx.TitlePairCache.js

```
@@ -14,8 +14,10 @@
  * @param {Object} config Configuration
  */
 mw.cx.TitlePairCache = function CXTitlePairCache( config ) {
+	var sourceLanguage = config.sourceLanguage;
```

ESLint

[error] Unexpected var, use let or const instead.

(no-var)